monai/transforms/inverse.py

```diff
@@ -63,48 +118,29 @@ class InvertibleTransform(Transform):
 
     """
 
-    def push_transform(
-        self, data: dict, key: Hashable, extra_info: Optional[dict] = None, orig_size: Optional[Tuple] = None
-    ) -> None:
-        """Append to list of applied transforms for that key."""
-        key_transform = str(key) + InverseKeys.KEY_SUFFIX
-        info = {InverseKeys.CLASS_NAME: self.__class__.__name__, InverseKeys.ID: id(self)}
-        if orig_size is not None:
-            info[InverseKeys.ORIG_SIZE] = orig_size
-        elif hasattr(data[key], "shape"):
-            info[InverseKeys.ORIG_SIZE] = data[key].shape[1:]
-        if extra_info is not None:
-            info[InverseKeys.EXTRA_INFO] = extra_info
-        # If class is randomizable transform, store whether the transform was actually performed (based on `prob`)
-        if isinstance(self, RandomizableTransform):
-            info[InverseKeys.DO_TRANSFORM] = self._do_transform
-        # If this is the first, create list
-        if key_transform not in data:
-            data[key_transform] = []
-        data[key_transform].append(info)
-
-    def check_transforms_match(self, transform: dict) -> None:
+    def check_transforms_match(self, transform: Mapping) -> None:
         """Check transforms are of same instance."""
-        if transform[InverseKeys.ID] == id(self):
+        xform_name = transform.get(TraceKeys.CLASS_NAME, "")
+        xform_id = transform.get(TraceKeys.ID, "")
+        if xform_id == id(self):
             return
         # basic check if multiprocessing uses 'spawn' (objects get recreated so don't have same ID)
-        if (
-            torch.multiprocessing.get_start_method() in ("spawn", None)
-            and transform[InverseKeys.CLASS_NAME] == self.__class__.__name__
-        ):
+        if torch.multiprocessing.get_start_method() in ("spawn", None) and xform_name == self.__class__.__name__:
             return
-        raise RuntimeError("Should inverse most recently applied invertible transform first")
+        raise RuntimeError(f"Error inverting the most recently applied invertible transform {xform_name} {xform_id}.")
+
+    def get_most_recent_transform(self, data: Mapping, key: Hashable = None):
+        """alias of self.peek_transform"""
+        return self.peek_transform(data, key)
 
-    def get_most_recent_transform(self, data: dict, key: Hashable) -> dict:
+    def peek_transform(self, data: Mapping, key: Hashable = None):
```

Comment:
I feel we don't need to add this function and use `get_most_recent_transform` as adapter, why not keep using `get_most_recent_transform` for this logic?

Thanks.

Reply:
that's to be consistent with pop/push , but I guess it's ok to drop this one